students/stefanjp1/lesson04/mailroom.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+def donor_names():
+    """Return a list of donor names."""
+    return donor_db.keys()
```

Comment:
note that it technically doesn't return a `list` data type; alternatively you can return -> `return list(donor_db)`